P202-Training.Domain/RoleService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using P202.Training.Data.Entities;
+using P202.Training.Data.Repositories;
+using System.Collections.Generic;
+using P202.Training.Data;
+using AutoMapper;
+
+namespace P202.Training.Domain
+{
+    public class RoleService : IRoleService
+    {
+        private RoleRepository _repository;
+        public void CreateRole(Role role)
+        {
+            if (role == null)
+                return;
+            Mapper.Initialize(cfg => cfg.CreateMap<Role, Role>());
+            var mapRole = Mapper.Map<Role>(role);
+            var roleRepository = new RoleRepository(new NHibernateSessionManager());
+            roleRepository.CreateRole(mapRole);
+        }
+
+        public bool DeleteRole(int roleId)
+        {
+            //Mock data
```

Comment:
You can add a TODO comment if you're leaving this implementation for another moment. Something like `// TODO: implemente DeleteRole method`